### PR TITLE
comm reassign cn ip for rackconnnect

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8e0f75179b/how-to/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggestions here: 
[reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2866269/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5c171adf2b/how-to/).

File: content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- Wait 20 to 30 minutes for the IP address to be released back into the
+  allocation pool.
+
+
+### Create a new port
+
+Perform the following steps to create a new port:
+
+- Use the Pitchfork call, [Create port](https://pitchfork.rax.io/networks/#create_port-networks)
```
Comment (RackerKate) [inline content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md]: Suggest adding a period at the end for consistency.

File: content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  `subnet_id`, and desired `ip_address`.
+
+- Submit the call.
+
+
+### Attach the Cloud Network
+
+Perform the following steps to attach the Cloud Network:
+
+- Log into the [Cloud Control Panel](https://login.rackspace.com).
```
Comment (RackerKate) [inline content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md]: Suggest "log in to"

File: content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+You can collect your Cloud Server UUIDs from the **Server Details** page in the
+[Cloud Control Panel](https://login.rackspace.com). In the top navigation bar,
+click **Select a Product > Rackspace Cloud**. Then select **Servers > Cloud
+Servers** and choose the server that you want to use.
+
+### Detach the Cloud Network
+
+Perform the following steps to detach the Cloud network:
+
+- Log into the [Cloud Control Panel](https://login.rackspace.com).
```
Comment (RackerKate) [inline content/rackconnect/reassign-a-cloud-network-ip-address-to-a-rackconnect-v3-cloud-server.md]: Suggest "Log in to"

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-29fcdafc52/how-to/).

**Pull request merged**